concordium-std/src/test_infrastructure/trie.rs

```diff
@@ -17,6 +17,16 @@ pub struct StateTrie {
     nodes:         Node,
     next_entry_id: Cell<StateEntryId>,
     entry_map:     RefCell<HashMap<StateEntryId, Vec<Index>>>,
+    iterators:     RefCell<BTreeMap<Vec<usize>, Iter>>,
+}
+
+/// Default constructor for a new state trie.
+/// 
+/// Added after clippy warning.
```

Comment:
That's not a good comment :) It does not provide value to the reader.

Instead document what the default value is, i.e., the empty state trie.